app.go

```diff
@@ -21,6 +26,23 @@ func NewApplication(l log.Logger, s stats.Statter) *Application {
 	}
 }
 
+func (a *Application) AWSStatus(loc l10n.LocaleInstance, area string, region string) (ApplicationResponse, error) {
+	title := loc.GetAny(loca.AWSStatusTitle)
```

Comment:
One must wonder if this belongs in here. It does not really constitute application logic. It is difficult to tell.

Reply:
Hmm, where else would we link the response to the translation then? in our `lambda.go`? Rather not, because its responsibility is just wrapping the request/response stuff.

I thought of it like this: I want to change logic of my intent: the response or the order of messages -> that all belongs to the app, IMHO.